src/libs/ReportUtils.js

```diff
@@ -377,11 +377,13 @@ function canShowReportRecipientLocalTime(personalDetails, report) {
     const hasMultipleParticipants = participantsWithoutExpensifyEmails.length > 1;
     const reportRecipient = personalDetails[participantsWithoutExpensifyEmails[0]];
     const reportRecipientTimezone = lodashGet(reportRecipient, 'timezone', CONST.DEFAULT_TIME_ZONE);
+    const isReportParticipantValidated = lodashGet(reportRecipient, 'validated', false);
```

Comment:
NAB: Can't we just skip declaring this as a variable and call the function directly in the return statement like we do with `!isChatRoom(report)`?

Reply:
We don't really call `lodashGet` anywhere inside of the actual condition so I wanted to avoid that and declare a variable similar to the other variables that are already declared with `lodashGet`.